package.json

```diff
@@ -46,7 +46,7 @@
     "jws": "^4.0.0",
     "lodash": "^4.17.21",
     "query-string": "^7.1.3",
-    "stellar-sdk": "^11.0.0-beta.6",
+    "stellar-sdk": "^11.1.0",
```

Comment:
@acharb @piyalbasu Could we please make it `@stellar/stellar-sdk` instead? Sorry I just noticed this little detail now